app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -183,8 +183,6 @@ object ComponentRegistry {
         TAB_TITLE_COMPOSE composeStory { net.skyscanner.backpack.demo.compose.BarChartStory() },
       ),
     ),
-    "Bottom Nav" story NodeData { BottomNavStory of R.layout.fragment_bottom_nav },
```

Comment:
A merge error was causing the screenshots to fail. Also, the screenshots were taken on the wrong emulator (you can see it in the diff).